main/src/main/scala/sbt/plugins/ScriptedPlugin.scala

```diff
@@ -43,9 +43,15 @@ object ScriptedPlugin extends AutoPlugin {
   }
 
   import autoImport._
+
+  override lazy val globalSettings = Seq(
+    scriptedBufferLog := true,
+    scriptedSbt := (sbtVersion in pluginCrossBuild).value,
```

Comment:
This is a breaking semantic change because most users don't set `sbtVersion in pluginCrossBuild` in Global scope, they set it in some project scope (like the root project scope).`

(If you think this is wrong and should be fixed, please vote for #2899)

Reply:
See https://github.com/sbt/sbt/blob/v1.1.0/main/src/main/scala/sbt/PluginCross.scala#L46-L47:

```scala
        state.log.info(s"Setting `sbtVersion in pluginCrossBuild` to $version")
        val add = List(sbtVersion in GlobalScope in pluginCrossBuild :== version) ++
```

Reply:
There are 78 other users of `sbtVersion in pluginCrossBuild` (on GitHub) that probably aren't doing that though...

https://github.com/search?l=Scala&q=%22sbtVersion+in+pluginCrossBuild%22&type=Code&utf8=%E2%9C%93

Reply:
The majority of the users I think are just reading the value, so they should be ok, but I am happy to move `scriptedSbt` back into project scoping.